recipes/px4tools/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "px4tools" %}
+{% set version = "0.4.5" %}
```

Comment:
@jakirkham I'm the author too, I just kept incrementing the build tags as I fixed issues related to conda forge packaging. I thought it was better than the alternative of overwriting tags. If you know a better way, let me know. I just uploaded the sdist to pypi.